File: challenges/codeeval/easy/092/Skhorn.py
#!/usr/bin/python

import sys

class penultimate_word:

	words_array = []
	new_word = ""
	count = 0
	with open(sys.argv[1], 'r') as test_cases:
		for line in test_cases:
			words_array = line.split(' ')
			print words_array[len(words_array)-2]
		#for word in words_array:
			


x = penultimate_word()

File: challenges/codeeval/easy/111/Skhorn.rb
#!/usr/bin/ruby
class LongestWord
   #@@no_of_customers=0
   def initialize()
      len = Array.new
      File.open(ARGV[0]).each_line do |line|
         #line = line.gsub(/  /, " @ ")
         words = line.split(' ').map { |x| x }
         for word in words
            len.push(word.length)
         end
         
         print words[len.index(len.max)]
         puts
         len = Array.new
      end 
   end
end

x = LongestWord.new()